Packs/Core/Playbooks/playbook-Suspicious_process_execution_by_scheduled_task_on_a_sensitive_server.yml

```diff
@@ -0,0 +1,893 @@
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#error#':
+      - "37"
+      '#none#':
+      - "40"
+    scriptarguments:
+      commands:
+        simple: powershell -Command "$ActionPath = '${alert.targetprocesscmd.[0]}'; $tasks = Get-ScheduledTask | Where-Object { $_.Actions | Where-Object { $_.Execute -eq $ActionPath } }; if ($tasks -or $tasks.Count -gt 0) { $tasks | ForEach-Object { Disable-ScheduledTask -TaskPath $_.TaskPath -TaskName $_.TaskName; Write-Host 'The task ' + $_.TaskName + ' has been disabled successfully.' } } else { Write-Host 'No tasks found running the action at ' + $ActionPath }"
+      endpoint_ids:
+        simple: ${alert.agentid}
+      timeout:
+        simple: "120"
+      timeout_in_seconds:
+        simple: "120"
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: errorPath
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1200
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "27":
+    id: "27"
+    taskid: c949acc9-c497-4818-8560-69c5c4044f39
+    type: condition
+    task:
+      id: c949acc9-c497-4818-8560-69c5c4044f39
+      version: -1
+      name: Check if the causality was blocked by the agent
+      description: Checks the incident's alerts for an alert that blocked the causality using the agent.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "29"
+      Blocked:
+      - "13"
+    separatecontext: false
+    conditions:
+    - label: Blocked
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: foundIncidents.CustomFields
+                filters:
+                - - operator: containsGeneral
+                    left:
+                      value:
+                        simple: foundIncidents.CustomFields.cid
+                      iscontext: true
+                    right:
+                      value:
+                        simple: alert.actionprocessinstanceid
+                      iscontext: true
+                    ignorecase: true
+                accessor: action
+            iscontext: true
+          right:
+            value:
+              simple: BLOCKED
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1870
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "29":
+    id: "29"
+    taskid: a6a1e05b-54c2-4fbd-891c-4089c958040d
+    type: regular
+    task:
+      id: a6a1e05b-54c2-4fbd-891c-4089c958040d
+      version: -1
+      name: Terminate Causality - Action Process
```

Comment:
We generally call it "Terminate Causality (CGO) ..."
you omitted the CGO by purpose?

Reply:
In this case, the causality is always svchost.exe. Therefore, I want to terminate the action process, which is the suspicious process.